beacon-light-client/solidity/contracts/balance_verifier/BalanceVerifierDiva.sol

```diff
@@ -69,6 +88,10 @@ contract BalanceVerifierDiva is BalanceVerifier, IBalanceVerifierDiva {
     );
   }
 
+  function setAccumulator(address _accumulator) external override onlyOwner {
+    ACCUMULATOR = _accumulator;
+  }
+
```

Comment:
setAccumulator must also set the branch

Reply:
Also, in what cases would you want the ability to set the accumulator. I think this code should execute in the constructor